scrapyd_k8s/logging_config.py

```diff
@@ -1,15 +1,15 @@
 import logging
 import sys
 
-def setup_logging(logging_level):
-    if not logging_level:
-        logging_level = 'INFO'  # Default to INFO if logging_level is None
+def setup_logging(log_level):
+    if not log_level:
+        log_level = 'INFO'  # Default to INFO if logging_level is None
 
-    level_name = str(logging_level).upper()
+    level_name = str(log_level).upper()
     numeric_level = logging.getLevelName(level_name)
     if not isinstance(numeric_level, int):
         raise ValueError(
-            f"Invalid logging level '{logging_level}'."
+            f"Invalid logging level '{log_level}'."
         )
     logging.basicConfig(
         level=numeric_level,
```

Comment:
Do you really need a numeric level here?
[`logging.setLevel`](https://docs.python.org/3.8/library/logging.html#logging.Logger.setLevel) accepts both number and string.